buildSrc/src/main/kotlin/BuildInfo.kt

```diff
@@ -97,7 +97,7 @@ open class BuildInfo(project: Project) {
 
   val hasMuslToolchain: Boolean by lazy {
     // see "install musl" in .circleci/jobs/BuildNativeJob.pkl
-    File(System.getProperty("user.home"), "staticdeps/bin/musl-gcc").exists()
+    File(System.getProperty("user.home"), "staticdeps/bin/x86_64-linux-musl-gcc").exists()
```

Comment:
FYI I used “staticdeps/bin/musl-gcc” because according to a comment in BuildNativeJob.pkl, that’s the path expected by GraalVM.